.github/workflows/build-rdo-package.yml

```diff
@@ -8,6 +8,7 @@ on:
   push:
     tags:
       - v*
+  workflow_dispatch:
```

Comment:
Just to be sure; it's supposed to be an empty object, right?

Reply:
Yes, it makes it possible manually schedule the workflow which should make it easier to test things.